src/manage_sql.c

```diff
@@ -23728,14 +23712,89 @@ where_qod (int min_qod)
       "task_id",                                                              \
       KEYWORD_TYPE_STRING },                                                  \
     { "(SELECT cve FROM nvts WHERE oid = nvt)", "cve", KEYWORD_TYPE_STRING }, \
+    { "(SELECT value"                                                         \
+      " FROM report_host_details"                                             \
+      " WHERE report_host = (SELECT id"                                       \
+      "                      FROM report_hosts"                               \
+      "                      WHERE report = results.report"                   \
+      "                      AND host = results.host)"                        \
+      " AND name = 'detected_by'"                                             \
+      " AND source_name = results.nvt)",                                      \
```

Comment:
This appears to be the iterator column that caused the SQL error for me, so it probably needs some limit and/or other criteria.